boring-sys/Cargo.toml

```
@@ -27,5 +27,5 @@ include = [
 ]
 
 [build-dependencies]
-bindgen = "0.57"
+bindgen = "0.58.1"
```

The latest version is now 0.59. I don't think we need to require later than .0, cargo will install the latest anyway and it gives our dependents more flexibility.
```suggestion
bindgen = "0.59"
```